README.md

```diff
@@ -27,6 +27,14 @@ joining.
 
 You can build and test the Java code from a Unix shell.
 
+Prerequisites:
+1. Download Java version 11
+2. Check what version of Java Maven is using: ```mvn --version```
+3. If Maven is using Java version 16:
+   - Open ```~/.bash_profile```
+   - Add ```export JAVA_HOME=<Path to your downloaded Java 11>```
+   - Save your change and run ```source ~/.bash_profile```
+
```

Comment:
After To run tests:, perhaps we can add:

  To build binary: ```mvn package```
  * This will produce `tool/target/datacommons-import-tool-1.0-SNAPSHOT-jar-with-dependencies.jar`

   To use the tool:  run ```java -jar <path-to-datacommons-import-tool-1.0-SNAPSHOT-jar-with-dependencies.jar>```
   * Its useful to create an alias like `alias dc-import='java -jar <path-to-datacommons-import-tool-1.0-SNAPSHOT-jar-with-dependencies.jar>'`, so you can invoke the tool as `dc-import`.

Reply:
done